package.json

```diff
@@ -87,9 +89,12 @@
     "docs:build": "docpress build",
     "commit": "git-cz",
     "cover": "istanbul cover test/index.js",
-    "lint": "eslint bin/ test/ lib/",
+    "lint": "eslint bin/ test/ lib/ .scripts/",
     "test": "npm run lint && node test | tap-spec",
-    "posttest": "rimraf .tmp"
+    "posttest": "rimraf .tmp",
+    "preinstall": "node .scripts/try_rename cached_node_modules node_modules",
```

Comment:
I'd be pretty concerned about using a preinstall script. `npm help scripts` has this to say:

```
       o Don't  use  install. Use a .gyp file for compilation, and prepublish for anything
         else. You should almost never have to explicitly  set  a  preinstall  or  install
         script.  If  you  are doing this, please consider if there is another option. The
         only valid use of install or preinstall scripts is for compilation which must  be
         done on the target architecture.
```

I'd also be afraid of using the `node` binary, which is some systems is `nodejs`. See my comment about publishing node_modules :)